lib/dependencies/NullDependency.js

```diff
@@ -29,6 +29,17 @@ NullDependency.Template = class NullDependencyTemplate extends DependencyTemplat
 	 * @returns {void}
 	 */
 	apply(dependency, source, runtimeTemplate, dependencyTemplates) {}
+
+	/**
+	 * @param {Dependency} dependency the dependency for which the template should be applied
+	 * @param {ReplaceSource} source the current replace source which can be modified
+	 * @param {RuntimeTemplate} runtimeTemplate the runtime template
+	 * @param {DependencyTemplates} dependencyTemplates the dependency templates
+	 * @returns {InitFragment[]}
+	 */
+	getInitFragments(dependency, source, runtimeTemplate, dependencyTemplates) {
```

Comment:
Better place this in Dependency.

We could allow `null` as return value to avoid array allocations.